packages/ui-react/src/pages/ScreenRouting/playlistScreens/PlaylistGrid/PlaylistGrid.tsx

```diff
@@ -30,7 +32,8 @@ const PlaylistGrid: ScreenComponent<Playlist> = ({ data, isLoading }) => {
     setFilter('');
   }, [data.feedid]);
 
-  const pageTitle = `${data.title} - ${config.siteName}`;
+  const title = (data?.[`title-${i18n.language}`] as string) || data.title;
```

Comment:
Do you think we could create a small function to create a translation key? Something like this:

```
export const useTranslationKey = (key: string) => {
   const { i18n } = useTranslation();
   
   return `${key}-${i18n.language}`
}
```
We could also reuse it for ShelfList component.

Reply:
It is not a blocker here, but could still be useful to have easier control over the translation key pattern in case additional change / logic is needed.

Reply:
I discussed it with @royschut and of course, because the hook only gives back a string it will only be called once. A hook is sufficient. I will add it!